models.go

```diff
@@ -148,6 +148,7 @@ type JWTClaims struct {
 	Country               string          `json:"country"`
 	ClientID              string          `json:"client_id"`
 	IsComply              bool            `json:"is_comply"`
+	StudioNamespace       string          `json:"studio_namespace"`
```

Comment:
Can we make it as `omitempty` field? In Core, we don't have this claims and it could be confuse what this is for

Reply:
I think this struct is only used when extracting jwt claims. It's not used when generating token. The one that's used for generating token is in the IAM. 
So I think, adding omit empty will have no impact.
Do you think we still need to add it?

Reply:
anyway, I've added omitempty tag on the studio_namespace